rocky/rocky/settings.py

```diff
@@ -249,10 +249,15 @@
     DATABASES["default"]["OPTIONS"] = {
         "sslmode": env("POSTGRES_SSL_MODE"),
         "sslrootcert": env.path("POSTGRES_SSL_ROOTCERT"),
-        "sslcert": env.path("POSTGRES_SSL_CERT"),
-        "sslkey": env.path("POSTGRES_SSL_KEY"),
     }
 
+    POSTGRES_SSL_CERT = env.path("POSTGRES_SSL_CERT", default="")
+    POSTGRES_SSL_KEY = env.path("POSTGRES_SSL_KEY", default="")
+
+    if POSTGRES_SSL_CERT and POSTGRES_SSL_KEY:
```

Comment:
Would it make sense to also set "sslmode" to "require" here when these options are configured?